packages/builder/src/index.ts

```diff
@@ -94,8 +95,15 @@ async function run(
     context.logger.info('All files pass linting.\n');
   }
 
+  const tooManyWarnings = maxWarnings >= 0 && totalWarnings > maxWarnings;
+  if (tooManyWarnings && printInfo) {
+    context.logger.error(
+      `Found too many warnings (maximum: ${options.maxWarnings}).`,
```

Comment:
It would be good to include the number that was found too, and give the user a hint on what to do next e.g.:

"Your configured maxWarnings is 100, but 123 warnings were found. Either increase your maxWarnings or fix some of the lint warnings."

Reply:
Sure, sounds good to me. My original intention was to mimic the output of the ESLint CLI, but this can be an improvement over it.